selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/workspaces/WorkspacesListTest.java

```diff
@@ -376,4 +386,38 @@ private void checkExpectedJavaWorkspaceDisplaying() {
   private int getWorkspacesCount() throws Exception {
     return testWorkspaceServiceClient.getAll().size();
   }
+
+  private List<Workspaces.WorkspaceListItem> getUnexpectedWorkspaces(
```

Comment:
these methods would be useful for the other tests and so more suitable for the **Workspaces** page object class